package.json

```diff
@@ -197,12 +199,14 @@
     "@metamask/forwarder": "^1.1.0",
     "@metamask/onboarding": "^0.2.0",
     "@sentry/cli": "^1.49.0",
-    "@storybook/addon-actions": "^5.2.6",
-    "@storybook/addon-info": "^5.1.1",
-    "@storybook/addon-knobs": "^5.1.1",
-    "@storybook/react": "^5.1.1",
+    "@storybook/addon-actions": "^5.2.8",
+    "@storybook/addon-info": "^5.2.8",
```

Comment:
Is this still required? It no longer appears to be referenced anywhere, since it was deleted from `.storybook/decorators.js`

Reply:
So we could take them out since we're not using them (as you mentioned), but we _might_ end up bringing these back? However, we can remove them from the `package.json` until then?

Reply:
Yeah, it'd be better to remove them for now if they're not used. It might help reduce the number of packages being added with this PR.